assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -12,117 +12,117 @@
+# Folder where Che stores internal data objects.
 che.database=${che.home}/storage
 
-# API service. Browsers initiate REST communications to Che server with this URL
+# API service. Browsers initiate REST communications to Che server with this URL.
 che.api=http://${CHE_HOST}:${CHE_PORT}/api
 
 # Che websocket major endpoint. Provides basic communication endpoint
-# for major websocket interaction/messaging.
+# for major websocket interactions and messaging.
 che.websocket.endpoint=ws://${CHE_HOST}:${CHE_PORT}/api/websocket
 
 # Che websocket minor endpoint. Provides basic communication endpoint
-# for minor websocket interaction/messaging.
+# for minor websocket interactions and messaging.
 che.websocket.endpoint_minor=ws://${CHE_HOST}:${CHE_PORT}/api/websocket-minor
 
-# Your projects are synchronized from the Che server into the machine running each
-# workspace. This is the directory in the ws runtime where your projects are mounted.
+# Projects are synchronized from the Che server into the machine running each
+# workspace. This is the directory in the workspace runtime where the projects are mounted.
 che.workspace.storage=${che.home}/workspaces
 
 # Your projects are synchronized from the Che server into the machine running each
 # workspace. This is the directory in the machine where your projects are placed.
 che.workspace.projects.storage=/projects
 
-# Used when devfile k8s/os type components requests project PVC creation
-# (applied in case of unique and perWorkspace PVC strategy. In case of common PVC strategy,
-# it will be rewritten with value of che.infra.kubernetes.pvc.quantity property)
+# Used when Kubernetes or OpenShift-type components in a devfile request project PVC creation
+# (Applied in case of 'unique' and 'per workspace' PVC strategy. In case of the 'common' PVC strategy,
+# it is rewritten with the value of the `che.infra.kubernetes.pvc.quantity` property.)
 che.workspace.projects.storage.default.size=1Gi
 
 # Defines the directory inside the machine where all the workspace logs are placed.
-# The value of this folder should be provided into machine e.g. like environment variable
-# so agents developers can use this directory for backup agents logs.
+# Provide this value into the machine, for example, as an environment variable.
+# This is to ensure that agent developers can use this directory to back up agent logs.
 che.workspace.logs.root_dir=/workspace_logs
 
-# Configures proxies used by runtimes powering workspaces
+# Configures proxies used by runtimes powering workspaces.
 che.workspace.http_proxy=
 che.workspace.https_proxy=
 che.workspace.no_proxy=
 
-# By default, when users access to a workspace with its URL the workspace
-# automatically starts if it is stopped. You can set this to false to disable this.
+# By default, when users access a workspace with its URL, the workspace
+# automatically starts (if currently stopped). Set this to `false` to disable this behaviour.
```

Comment:
+1 for British/Canadian spelling of behaviour :D

Reply:
Force of habit, but unfortunately, AmEng it is for docs :/